github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/dsl/context/GitHubPRTriggerDslContext.java

```diff
@@ -33,6 +35,14 @@ public void setPreStatus() {
         setPreStatus = true;
     }
 
+    public void setCancelQueued() {
```

Comment:
I disagree, but will change it. I'd say  `preStatus` property is exposed as `setPreStatus` because of  the setter of `GitHubPRTrigger`, not "set" as verb. Either the DSL should be also just `preStatus` so there is just `preStatus` and `cancelQueued` or both would have the "set" prefix to set the property: `setPreStatus` and `setCancelQueued`. 

Anyway, changing it your proposal so there is `setPreStatus` which does `GitHubPRTrigger.setPreStatus(true)` and `cancelQueued` which does `GitHubPRTrigger.setCancelQueued(true)`.